pkg/setting/setting.go

```diff
@@ -388,6 +388,11 @@ func (cfg Cfg) IsNgAlertEnabled() bool {
 	return cfg.FeatureToggles["ngalert"]
 }
 
+// IsTrimDefaultsEnabled returns whether the standalone trim dashboard default feature is enabled.
+func (cfg Cfg) IsTrimDefaultsEnabled() bool {
```

Comment:
This looks good, but is not used.

Reply:
It is used in schemaloader in another file 
https://github.com/grafana/grafana/pull/33465/files?file-filters%5B%5D=.go&file-filters%5B%5D=.json#diff-b03c5e2fd0f9363b9e2e90700b1b5654b425c4b574e528f684a4041ca2615221R57